components/assetcard/index.css

```
@@ -13,52 +13,84 @@ governing permissions and limitations under the License.
-  --spectrum-assetcard-selectionindicator-margin: var(--spectrum-global-dimension-size-150);
-  --spectrum-assetcard-selectionindicator-size: var(--spectrum-global-dimension-static-size-500);
-  --spectrum-assetcard-selectionindicator-border-radius: var(--spectrum-alias-border-radius-regular);
-
-  --spectrum-assetcard-heading-text-color: var(--spectrum-global-color-gray-900);
-  --spectrum-assetcard-content-text-color: var(--spectrum-global-color-gray-700);
+  --spectrum-assetcard-border-color-hover: var(--spectrum-gray-500);
+  --spectrum-assetcard-border-color-down: var(--spectrum-gray-600);
+
+  --spectrum-assetcard-focus-ring-gap: 5px;
+  --spectrum-assetcard-focus-indicator-thickness: var(--spectrum-focus-indicator-thickness);
+
+  /* selection indicator - checkbox or number */
+  --spectrum-assetcard-selectionindicator-background-color-default: rgba(var(--spectrum-gray-100-rgb), 0.9);
+  --spectrum-assetcard-selectionindicator-size: var(--spectrum-card-selection-background-size);
+  --spectrum-assetcard-selectionindicator-border-radius: var(--spectrum-corner-radius-100);
+  --spectrum-assetcard-selectionindicator-offset-y: 4px;
+  --spectrum-assetcard-selectionindicator-blur: 6px;
+  --spectrum-assetcard-selectionindicator-color: var(--spectrum-white);
+  --spectrum-assetcard-selectionindicator-font-weight: var(--spectrum-bold-font-weight);
+  --spectrum-assetcard-selectionindicator-font-size: var(--spectrum-font-size-400);
+
+  /* title */
+  --spectrum-assetcard-title-text-color: var(--spectrum-gray-900);
+  --spectrum-assetcard-title-font-family: var(--spectrum-sans-font-family-stack);
+  --spectrum-assetcard-title-font-weight: var(--spectrum-heading-sans-serif-font-weight);
+  --spectrum-assetcard-title-font-style: var(--spectrum-default-font-style);
+  --spectrum-assetcard-title-line-height: var(--spectrum-line-height-100);
+  --spectrum-assetcard-title-letter-spacing: 0;
+
+  /* header content */
+  --spectrum-assetcard-header-content-text-color: var(--spectrum-gray-900);
+  --spectrum-assetcard-header-content-font-family: var(--spectrum-sans-font-family-stack);
+  --spectrum-assetcard-header-content-font-weight: var(--spectrum-body-sans-serif-font-weight);
+  --spectrum-assetcard-header-content-font-style: var(--spectrum-default-font-style);
+  --spectrum-assetcard-header-content-line-height: var(--spectrum-line-height-200);
+  --spectrum-assetcard-header-content-letter-spacing: 0;
+
+  /* content */
+  --spectrum-assetcard-content-text-color: var(--spectrum-gray-700);
+  --spectrum-assetcard-content-font-family: var(--spectrum-sans-font-family-stack);
+  --spectrum-assetcard-content-font-weight: var(--spectrum-body-sans-serif-font-weight);
+  --spectrum-assetcard-content-font-style: var(--spectrum-default-font-style);
+  --spectrum-assetcard-content-line-height: var(--spectrum-line-height-200);
+  --spectrum-assetcard-content-letter-spacing: 0;
+
+  --spectrum-assetcard-content-margin-block-start: var(--spectrum-spacing-75);
```

```suggestion
  --spectrum-assetcard-content-margin-block-start: calc(
		var(--spectrum-assetcard-title-font-size) *
			var(--spectrum-heading-margin-bottom-multiplier)
	);
```

I'd like to see us capture the logic here which is pulled from the updated typography styles.

---

Currently this value is 4px regardless of text size, changing to the suggested syntax would change the margin for Large scale to 4.25px. Do we want to introduce this change?